<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pbtn_output">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>输出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_frp">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>映射</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_about">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>关于</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_visit_help">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>帮助</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_upd_set">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="15" colspan="3">
        <widget class="QLineEdit" name="motd"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="n_1">
         <property name="text">
          <string>服务端路径：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="8" colspan="4">
        <widget class="QLineEdit" name="max_player">
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>GB</string>
         </property>
        </widget>
       </item>
       <item row="6" column="15" colspan="3">
        <widget class="QLineEdit" name="server_port">
         <property name="text">
          <string>19132</string>
         </property>
        </widget>
       </item>
       <item row="7" column="5" colspan="3">
        <widget class="QLabel" name="n_11">
         <property name="text">
          <string>是否启用命令方块：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="2" colspan="5">
        <widget class="QComboBox" name="cbox_using_java">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>Java17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Java16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Java8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>系统变量</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="10">
        <widget class="QLabel" name="n_5">
         <property name="text">
          <string>GB</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="18">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtn_ok_adv_set">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="12" colspan="3">
        <widget class="QLabel" name="n_9">
         <property name="text">
          <string>服务器端口：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="13" rowspan="2" colspan="5">
        <widget class="QPushButton" name="pbtn_download_server">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>下载</string>
         </property>
        </widget>
       </item>
       <item row="5" column="7" colspan="3">
        <widget class="QSpinBox" name="max_ram">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="4">
        <widget class="QComboBox" name="cbox_pvp">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>True</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>False</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="n_10">
         <property name="text">
          <string>开启 PVP:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="15">
        <widget class="QLabel" name="lb_path">
         <property name="text">
          <string>None</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="12" colspan="3">
        <widget class="QLabel" name="n_12">
         <property name="text">
          <string>服务器列表界面提示：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="11" colspan="7">
        <widget class="QPushButton" name="pbtn_dis_log4j2">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>通过启动参数禁用Log4j2</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5" colspan="2">
        <widget class="QLabel" name="n_4">
         <property name="text">
          <string>最大内存：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="n_8">
         <property name="text">
          <string>使用Java:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="17">
        <widget class="QPushButton" name="pbtn_select_path">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>选择路径</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QSpinBox" name="min_ram">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="n_3">
         <property name="text">
          <string>最小内存：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="5" colspan="3">
        <widget class="QLabel" name="n_6">
         <property name="text">
          <string>最大玩家数量：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="8" colspan="4">
        <widget class="QComboBox" name="cbox_command_block">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>True</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>False</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0" colspan="18">
        <widget class="QPushButton" name="pbtn_start_server">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>开启服务器</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6" rowspan="2" colspan="7">
        <widget class="QPushButton" name="pbtn_show_java_path">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>点击显示当前路径</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="n_7">
         <property name="text">
          <string>正版验证：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="4">
        <widget class="QComboBox" name="cbox_mojang">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>True</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>False</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="18">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="n_13">
            <property name="text">
             <string>以下设置将在服务器重启后生效</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
